avr-specs/avr-atmega328pb.json

```diff
@@ -0,0 +1,28 @@
+{
+    "arch": "avr",
+    "atomic-cas": false,
+    "cpu": "atmega328pb",
```

Comment:
@Rahix looks like I cannot with this spec. Not sure how to solve this problem

Reply:
@jkristell, you originally added ATmega328PB support.  What target spec were you compiling with?

Reply:
I used the atmega328p spec.

Reply:
Thanks!  @explicite, in that case just drop the custom spec and use `avr-atmega328p.json` in the CI.